data/libs/utils.lua

```diff
@@ -206,4 +206,47 @@ utils.inherits = function (baseClass, name)
 	return new_class
 end
 
+utils.print_r = function(t)
```

Comment:
it recursively prints tables, so you can look at the contents. useful for debugging for me :)